language/tools/move-cli/src/main.rs

```diff
@@ -750,11 +748,11 @@ fn main() -> Result<()> {
         Command::Publish {
             source_files,
             no_republish,
-            dry_run,
         } => {
             move_args.prepare_mode(true)?;
             let state = publish(&move_args, !*no_republish, source_files)?;
-            maybe_commit_effects(&move_args, !*dry_run, None, &state)
+            let dry_run = false; // publish is the same as check if dry_run is set
+            maybe_commit_effects(&move_args, dry_run, None, &state)
```

Comment:
Would it be better if we just have `maybe_commit_effects(&move_args, false, None, &state)` here?

Reply:
It would be shorter, but I find code with anonymous boolean literals hard to read/update. Ideally, Rust would have named parameters to help us with this...